roles/postgresql/handlers/main.yml

```
@@ -14,6 +14,12 @@
   delegate_to: '{{ postgres_host }}'
   when: not postgresql_is_local
 
+- name: reload postgres
```

Adding `local` would make the distinction between the two handlers clear - either we reload the remote postgres, or we reload the local postgres:
```suggestion
- name: reload local postgres
```